cli/lock.ml

```diff
@@ -532,6 +557,18 @@ let require_cross_compile =
     (fun x -> `Require_cross_compile x)
     Arg.(value & flag & info ~doc [ "require-cross-compile" ])
 
+let minimal_update =
+  let doc =
+    "Generate a lock file with minimum dependency changes compared to the \
+     previous lock file. It can remove or add packages based on the dependency \
+     specification in the local opam files but will not upgrade or downgrade \
+     previously locked packages unless strictly necessary.\n\
+     Require a lock file to exist in the target lock file path."
```

Comment:
I feel like specifying the mechanism how it does so e.g. "prefers to keep the versions as in the existing lock file unless strictly necessary" would make it feel a bit less magical - then it is clear how it tries to do minimal dependency changes without complicating the description too much.

What do you think?

Reply:
Yeah that's shorter and clearer! Let's change it!